apps/web/src/pages/TalentNominations/NominationGroup/components/TalentNominationAccordionItem.tsx

```diff
@@ -0,0 +1,559 @@
+            "Lateral movement<hidden> nomination declined</hidden>",
+          id: "2VqFMs",
+          description:
+            "Status description when the nomination doest not include the lateral movement option",
+        });
+
+  const developmentProgramsStatusDescription =
+    talentNomination.nominateForDevelopmentPrograms
+      ? defineMessage({
+          defaultMessage:
+            "Development programs<hidden> nomination accepted</hidden>",
+          id: "13C3j8",
+          description:
+            "Status description when the nomination includes the development programs option",
+        })
+      : defineMessage({
+          defaultMessage:
+            "Development programs<hidden> nomination declined</hidden>",
+          id: "DRrUMl",
+          description:
+            "Status description when the nomination doest not include the development programs option",
+        });
+
+  const advancementReferenceIsAUser = !!talentNomination.advancementReference;
+
+  const lateralMovementOptionValuesInThisNomination =
+    talentNomination.lateralMovementOptions?.map((option) => option.value) ??
+    [];
+
+  // I don't actually need a set of options but the sort is handy
+  const lateralMovementListItems = localizedEnumToOptions(
+    options?.talentNominationLateralMovementOptions,
+    intl,
+    [
+      TalentNominationLateralMovementOption.SmallDepartment,
+      TalentNominationLateralMovementOption.LargeDepartment,
+      TalentNominationLateralMovementOption.CentralDepartment,
+      TalentNominationLateralMovementOption.NewDepartment,
+      TalentNominationLateralMovementOption.ProgramExperience,
+      TalentNominationLateralMovementOption.PolicyExperience,
+      TalentNominationLateralMovementOption.Other,
+    ],
+  ).map((option) => ({
+    key: option.value,
+    value: lateralMovementOptionValuesInThisNomination.includes(
+      option.value as TalentNominationLateralMovementOption,
+    ),
+    label: option.label,
+  }));
+
+  const developmentProgramIdsInThisNomination =
+    talentNomination.developmentPrograms?.map((program) => program.id) ?? [];
+
+  const developmentProgramListItems =
+    talentNomination.talentNominationEvent.developmentPrograms?.map(
+      (program) => ({
+        key: program.id,
+        value: developmentProgramIdsInThisNomination.includes(program.id),
+        label:
+          program.name?.localized ??
+          intl.formatMessage(commonMessages.notFound),
+      }),
+    ) ?? [];
+
+  // OTHER is not a real development program so we have to add a fake option
+  developmentProgramListItems.push({
+    key: "OTHER",
+    value: !!talentNomination.developmentProgramOptionsOther,
+    label: intl.formatMessage(commonMessages.other),
+  });
+
+  const skillListItems =
+    unpackMaybes(talentNomination.skills).map((skill) => ({
+      key: skill.id,
+      label:
+        skill.name.localized ?? intl.formatMessage(commonMessages.notFound),
+    })) ?? [];
+  skillListItems.sort();
```

Comment:
Is this actually doing anything? 🤔 I would expect something like:

```tsx
skillListItems.sort((a, b) => a.label.localeCompare(b.label));
```

Reply:
:facepalm: I'm not sure what I was thinking there.  Thanks for catching that.
[fix skills sorting](https://github.com/GCTC-NTGC/gc-digital-talent/pull/13099/commits/6720e80b1aab9dbc94c958076b380044a5da98a1)